### ISSUE TITLE
Module '"chai-exclude"' has no default export.ts

### ISSUE THREAD

**Issue by jonsalvas** [mesaugat/chai-exclude#22]
First: Thanks for providing this useful plugin. 

Unfortunately I cannot use it in typescript. When trying to import the module via the code from your documentation
```
import * as chai from 'chai';
import chaiExclude from 'chai-exclude';

chai.use(chaiExclude);
```
Typescript complains about the default export of chai-exclude:
_Module '"chai-exclude"' has no default export.ts_

**Comment by mesaugat** [comment]
Hello @jonsalvas, which version of `chai-exclude` are you using? This should have been fixed is version 2.0.0

**Comment by jishei-128** [comment]
Hi @mesaugat, I do not know which version I used before but with 2.0.1 it is working fine. Thanks!
